src/CallHandler.js

```diff
@@ -416,10 +416,10 @@ function _startCallApp(roomId, type) {
 
     const currentRoomWidgets = WidgetUtils.getRoomWidgets(room);
 
-    if (WidgetEchoStore.roomHasPendingWidgetsOfType(room, currentRoomWidgets, 'jitsi')) {
+    if (WidgetEchoStore.roomHasPendingWidgetsOfType(roomId, currentRoomWidgets, 'jitsi')) {
         const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
 
-        Modal.createTrackedDialog('Already have pending Jitsi Widget', '', ErrorDialog, {
+        Modal.createTrackedDialog('Call being placed', '', ErrorDialog, {
```

Comment:
On paper this looks like we're tracking a call being placed. I would suggest "Call already in progress".